api/src/routes/api.ts

```diff
@@ -29,7 +59,7 @@ router.get("/:category", function (req, res) {
     res.json(category.raw());
 });
 
-router.get("/:category/:job", function (req, res) {
+router.get("/:category/:job", requireToken, function (req, res) {
```

Comment:
Should we apply requireToken to all the API routes before merging, now that we've proven the concept?